client/src/pages/ErrorPage/ErrorPage.js

```diff
@@ -1,5 +1,39 @@
 import React from 'react';
+import { Box, Typography } from '@mui/material';
+import Button from '@mui/material/Button';
+import { useNavigate } from 'react-router-dom';
+import styled from '@emotion/styled';
 
-const ErrorPage = () => <div>Error</div>;
+const ErrorContainer = styled('div')({
```

Comment:
I would like to suggest adding safe paddings on left and right so text would not be sticked to borders